backend/actions/event.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import { Event } from "../models/Event";
+import mongoose from "mongoose";
+
+export const AddEvent = async (req: Request, res: Response) => {
+  const { email, time, eventName, description } = req.body;
+
+  try {
+    const result = await new Event({
+      email,
+      time,
+      eventName,
+      description,
+    }).save();
+    res.send({ res: "Success", data: result });
```

Comment:
It's not required but it would be better to specify the status code in the responses.
```suggestion
    res.status(200).json({ isSuccess: true, data: result });
```